stacktrace.go

```diff
@@ -213,11 +206,7 @@ func filterFrames(frames []Frame) []Frame {
 			continue
 		}
 		// sentry internal frames
-		isTestFile := isTestFileRegexp.MatchString(frame.AbsPath)
-		isExampleFile := isExampleFileRegexp.MatchString(frame.AbsPath)
-		if strings.Contains(frame.AbsPath, "github.com/getsentry/sentry-go") &&
-			!isTestFile &&
-			!isExampleFile {
+		if frame.Module == "github.com/getsentry/sentry-go" {
```

Comment:
It was a bit curious to me that a `Frame` has both a `Module` and a `Package` field. What do they mean exactly? I might have missed it, but it seemed like the `Package` field is never set. @kamilogorek for help 🆘

Reply:
See: https://docs.sentry.io/development/sdk-dev/event-payloads/stacktrace/#frame-attributes